### ISSUE TITLE
Unity Multiplayer Virtual Reality with Normcore

### ISSUE THREAD

**Issue by jeffrafter** [jeffrafter/jeffrafter.github.io#42]
![image](https://user-images.githubusercontent.com/4064/104861763-21171e00-58ff-11eb-80ed-6092a119f4bb.png)

Questions and comments for https://jeffrafter.com/unity-multiplayer-vr-with-normcore/

**Comment by coolguy770** [comment]
how would I create lobby system that uses room codes to separate a large amount of players?

**Comment by trlatimer** [comment]
I found this very helpful! Definitely explained things well and I will have to look more into Normcore as it was impressively simple to setup.  However, a lot of the images were not showing for me. I tried a few different browsers but was unable to view them. Most weren't that big of a deal but the image for the offset values within the Follow script definitely added some difficulty.

**Comment by tleylan** [comment]
The broken image links are definitely an issue. From what I can tell they are fetched from a service somewhere https://rpl.cat/uploads/8E-WEWIob3jKUXnQKRYTW4FOYneniX3PZ-VAxP4yKEo/public.png which isn't responding.

**Comment by tleylan** [comment]
Oh I forgot... thanks for writing up the article BTW very handy (just need the images) :-)

**Comment by tleylan** [comment]
Serious question but did anyone follow this to the end and get everything to work? The text typically explains what would be in the images but in a few cases the images are the only source of a setting and we can't see them. I also got two obsolete method warnings (I adjusted those, correctly I believe). And a bunch of Burst compiler errors would occur. I finally disabled Burst and that should do it.

**Comment by jeffrafter** [comment]
This was made pre-Burst... it is very possibly out of date... Unity moves fast.

**Comment by tleylan** [comment]
I also see the final avatar standing next to my view when running which is clearly a setting error of some sort. Couldn't quite determine what gets set where in each of the Follow property settings.

Thanks Jeff, ain't that the truth! Any chance you can find time to adjust those images it would help a lot?

**Comment by jeffrafter** [comment]
Yeah, I can try to find time maybe this weekend. Cloudflare dropped support for .cat domains which is where the images were hosted... so now I find myself in the unfortunate position of needing to find a new CDN. Which is a big move...

**Comment by trlatimer** [comment]
I was able to get it working and matching my VR rig without modifications
to the code, just with experimenting with offsets. Only issues I’m running
into now is handling height, collision with my character controller on the
XR origin, and being able to extend my hands further than the model will go.

**Comment by tleylan** [comment]
That is good to hear. I think the instructions are well done but it would be great if they could be updated/supplemented as people follow along. If a bunch of people find a particular thing puzzling or hard to understand it typically means the explanation can be improved. I think my issues will probably disappear when I can see the images (any chance of a zip file perhaps?) but there are 1 or 2 things that didn't jive with what I was seeing.

**Comment by tleylan** [comment]
@jeffrafter is the complete project available anywhere? This would also give us a chance to compare what we have with what the settings are supposed to be.

**Comment by tleylan** [comment]
@trlatimer Perhaps you would be interested in comparing notes. If you have a Discord account that would work out well for me.

**Comment by trlatimer** [comment]
@tleylan Sure! My discord is Girdian#9740. Also, I finally had a chance to get work on this project again and realized that I haven't yet tied the model into multiplayer tracking. I have the model tracking to the hands for the client without issue and all I should have to do is just map the clients XR Origin if we are the ones being spawned in but there is always room for issues. I plan to work that in shortly and can let you know how that goes as well. I'm also using Photon rather than Normcore.

**Comment by jeffrafter** [comment]
Thanks for the patience all, I was finally able to fix the images on this post <3

**Comment by GauravGupta17** [comment]
does normcore sdk supports normcore , i have implemented the normcore , but i cant see the other avatar , i can only hear audio of other player

**Comment by jeffrafter** [comment]
Sorry can you clarify your question? Normcore does have support forums but if you can add some more detail I might be able to help debug...

**Comment by GauravGupta17** [comment]
Hi i have resolve the last issue , i am cardboard xr sdk , so i dont have any hand controller , i just to make two avatar face each other in runtime , if avatar are more than 2 they allign in circle , how can i achieve this ?

**Comment by biocaster** [comment]
Hi Jeff your guide was really excellent. However would be amazing if you can show how we can do damage to the other player with the gun and it's projectiles.

**Comment by jeffrafter** [comment]
Thanks @biocaster I'll do that... I plan on revisiting this and updating it.
